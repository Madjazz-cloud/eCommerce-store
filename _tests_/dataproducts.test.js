import getAllProducts, { getProductById } from '../dataproducts';

test('gets a Product by ID', () => {
  expect(getProductById('2')).toEqual(
    getAllProducts().find(product => {
      return product.id === '2';
    })
  );

  expect(getProductById('0')).toMatchSnapshot();
});

test('returns undefind when passed a non-id', () => {
  expect(getProductById('')).toBe(undefined);
});
