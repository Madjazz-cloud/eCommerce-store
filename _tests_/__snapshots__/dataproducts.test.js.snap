// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gets a Product by ID 1`] = `
Object {
  "description": "description 1",
  "id": "0",
  "name": "Product-1",
  "picture": "/static/logo.jpg",
  "price": "5,00 â‚¬",
}
`;
